desk/app/groups.hoon

```diff
@@ -1279,6 +1288,19 @@
       %+  weld  /server/(scot %p our.bowl)/[q.flag]
       /invite/retry/(scot %p ship)/(scot %ud retry)/(scot %dr delay)
     (emit [%pass wire %arvo %b %wait (add now.bowl delay)])
+  ::
+      ::  v6 -> v7 migrate group subscriptions
+      ::
+      ::  the group updates subscription path has changed,
+      ::  so we must re-subscribe on the new path post load.
+      ::
+      [%load %v7 %subscriptions ~]
+    =.  cor
+      %+  roll
+        ~(tap by groups)
+      |=  [[=flag:g [=net:g *]] =_cor]
+      go-abet:(go-safe-sub:(go-abed:go-core:cor flag) |)
+    cor
```

Comment:
We defer this because the `%leave`s from `+load` won't affect the bowl until end of event, and we don't want to wait til next reload for `+inflate-io` to get called, right? Why not just call `+inflate-io` here?

Reply:
It would be more robust to call `+inflate-io` as a general pattern, I will adjust it.